Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalIsoTrkFromAlCaRaw_cff.py

```diff
@@ -0,0 +1,20 @@
+import FWCore.ParameterSet.Config as cms
+
+#--------------------------------------------------------------------
+# AlCaReco filtering for running on dedicated AlCaRaw: AlCaHcalIsoTrk
+#--------------------------------------------------------------------
+
+from Calibration.HcalAlCaRecoProducers.alcaHcalIsotrkProducer_cff import *
```

Comment:
instead of importing and then changing the attributes (which creates "long-distance interactions"), I would suggest to do something like (python pseudo-code below)

```python
from Calibration.HcalAlCaRecoProducers.alcaHcalIsotrkProducer_cff import  alcaHcalIsotrkProducer as _alcaHcalIsotrkProducer

alcaHcalIsotrkFromAlCaRawProducer = _alcaHcalIsotrkProducer.clone(
    labelTrack = "hltMergedTracksSelector",
    ....
)

seqALCARECOHcalCalIsoTrkFromAlCaRaw = cms.Sequence(alcaHcalIsotrkFromAlCaRawProducer * alcaHcalIsotrkFilter)
```

Reply:
Implemented and reworked to do this cloning